src/System.Drawing.Common/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,6 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
```

Comment:
Please add licensing for these new files
```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.

﻿// This file is used by Code Analysis to maintain SuppressMessage
```